src/system.json

```diff
@@ -117,6 +124,62 @@
       "system": "rqg",
       "path": "assets/packs/name-tables.db",
       "type": "RollTable"
+    },
```

Comment:
@wake42  In my attempt at adding rqids (which maybe was more successful than I thought but because of my corrupted world compendium pack I reverted it all) I had sorted all of the packs in this file by name using an online JSON sorter.  I'm not going to do that again right now but I think it's a good idea.